### PR TITLE
Fixing Missing documentation bug issue #27

### PR DESCRIPTION
**Pull request by umarbrowser**
I just start writing documentation to the python and bash source.

### CODE REVIEW

File: bin/dev/plot_slice_surf.py [hunk 1]
```diff
@@ -1,4 +1,30 @@
 # -*- encoding: utf-8 -*-
+# Tools for building full brain network models from standard structural MR scans.
+# Web-UI helpful to run brain-simulations. To use it, you also need do download
+# TheVirtualBrain-Scientific Package (for simulators). See content of the
+# documentation-folder for more details. See also http://www.thevirtualbrain.org
+#
+# (c) 2012-2017, Baycrest Centre for Geriatric Care ("Baycrest") and others
```
Comment (liadomide) [inline bin/dev/plot_slice_surf.py]: This is not the correct Copyright text. If we decide to include it in all scripts, should be the text from https://github.com/the-virtual-brain/tvb-recon/blob/master/LICENSE.
But, for the moment, I see the other scripts do not have a copyright at top.

File: bin/dev/test_subaseg.py [hunk 2]
```diff
@@ -1,4 +1,34 @@
+# -*- coding: utf-8 -*-
+# Tools for building full brain network models from standard structural MR scans.
+# Web-UI helpful to run brain-simulations. To use it, you also need do download
+# TheVirtualBrain-Scientific Package (for simulators). See content of the
+# documentation-folder for more details. See also http://www.thevirtualbrain.org
+#
+# (c) 2012-2017, Baycrest Centre for Geriatric Care ("Baycrest") and others
```
Comment (liadomide) [inline bin/dev/test_subaseg.py]: See prev comment about copyright notice

Comment (umarbrowser) [reply bin/dev/test_subaseg.py]: ok

File: bin/dev/plot_slice_surf.py (hunk 1 above)
Comment (umarbrowser) [reply bin/dev/plot_slice_surf.py]: ok i will used that

Comment (liadomide) [reply bin/dev/plot_slice_surf.py]: NO, rather leave it as it was (no copyright text in scripts). We might add it only later.

**Comment by umarbrowser** [comment]
ok right away

**Comment by liadomide** [comment]
It is fine to add comments on the important functions in the package. 
But, we are not really after that in bug #27, especially if something close to it can be obtained from the function name.
Bug #27 should refer also to an improved Readme or docs folder with multiple text-based (actually .RST most probably) documents, explaining what this module is expected to do.

**Comment by umarbrowser** [comment]
ok i got it now. right away

**Comment by umarbrowser** [comment]
Ok i got it, Right way i will start👌

File: bin/dev/test_subaseg.py (hunk 2 above)
Comment (umarbrowser) [reply bin/dev/test_subaseg.py]: Can you please look again i remove all the license

**Comment by maedoc** [comment]
this PR has 22k changes in the diff😬something in Git-land has gone wrong 🤔

Umar I appreciate your effort but we need to find something better than docstrings for you to work on. Did you make progress on the packaging distribution topics?

**Comment by umarbrowser** [comment]
Yes sir i make Some progress while preparing for GSoC 2018 here are some,
Dockerfile https://github.com/umarbrowser/dockerImageTvb 

Conda packaging https://github.com/umarbrowser/staged-recipes

Neurodebian packaging https://github.com/umarbrowser/Neuropackage

Vagrantfile script https://github.com/umarbrowser/vagrantTvb

**Comment by umarbrowser** [comment]
Hi @maedoc i will be glad to have something to work on, just Waiting for it.

File: bin/dev/plot_slice_surf.py (hunk 1 above)
Comment (umarbrowser) [reply bin/dev/plot_slice_surf.py]: Ok Thanks, Right away

File: bin/dev/test_xfms.py [hunk 3]
```diff
@@ -1,4 +1,31 @@
...
+#   CITATION:
+# When using The Virtual Brain for scientific publications, please cite it as follows:
+#
+#   Paula Sanz Leon, Stuart A. Knock, M. Marmaduke Woodman, Lia Domide,
+#   Jochen Mersmann, Anthony R. McIntosh, Viktor Jirsa (2013)
+#       The Virtual Brain: a simulator of primate brain network dynamics.
+#   Frontiers in Neuroinformatics (7:10. doi: 10.3389/fninf.2013.00010)
+#
+#
+
```
Comment (liadomide) [inline bin/dev/test_xfms.py]: This should not be here

File: bin/label-centers.py [hunk 4]
```diff
@@ -1,14 +1,18 @@
...
 import argparse
 import logging
 import nibabel as nb
 import numpy as np
 
-
 def label_volume_centers(label_volume):
+    """Crete a volume center and insert numpy to calculate it
+       while debugging it. """
+
```
Comment (liadomide) [inline bin/label-centers.py]: This comment makes no sense. The function aims to generate tuples (label, x, y, z)

File: bin/label-centers.py [hunk 5]
```diff
@@ -1,14 +1,18 @@
 #!/usr/bin/env python3
+# -*- coding: utf-8 -*-
 
+# imports
```
Comment (liadomide) [inline bin/label-centers.py]: Not necessary ... it is obvious that the imports are here

**Comment by umarbrowser** [comment]
On Feb 23, 2018 12:22 PM, "Lia Domide" <notifications@github.com> wrote:
>
> @liadomide commented on this pull request.
>
> I tend to agree with Marmaduke. The direction for solving this does not
seem healthy.
>
I am sorry, going to fix it.
> ________________________________
>
> In bin/dev/test_xfms.py:
>
> > +# WARRANTY; without even the implied warranty of MERCHANTABILITY or
FITNESS FOR A
> +# PARTICULAR PURPOSE.  See the GNU General Public License for more
details.
> +# You should have received a copy of the GNU General Public License
along with this
> +# program.  If not, see <http://www.gnu.org/licenses/>.
> +#
> +#
> +#   CITATION:
> +# When using The Virtual Brain for scientific publications, please cite
it as follows:
> +#
> +#   Paula Sanz Leon, Stuart A. Knock, M. Marmaduke Woodman, Lia Domide,
> +#   Jochen Mersmann, Anthony R. McIntosh, Viktor Jirsa (2013)
> +#       The Virtual Brain: a simulator of primate brain network dynamics.
> +#   Frontiers in Neuroinformatics (7:10. doi: 10.3389/fninf.2013.00010)
> +#
> +#
> +
>
> This should not be here
i will remove it now.
>
> ________________________________
>
> In bin/label-centers.py:
>
> >  def label_volume_centers(label_volume):
> +    """Crete a volume center and insert numpy to calculate it
> +       while debugging it. """
> +
>
> This comment makes no sense. The function aims to generate tuples (label,
x, y, z)
>
ok i got it, i will try my best to understand the code
> ________________________________
>
> In bin/label-centers.py:
>
> >
> +# imports
>
> Not necessary ... it is obvious that the imports are here
>
Ok

File: bin/dev/test_xfms.py (hunk 3 above)
Comment (umarbrowser) [reply bin/dev/test_xfms.py]: ok

**Comment by liadomide** [comment]
I close this. Meanwhile, Paula added documentation in README

**Pull request closed without merging**